spec/models/form1095_b_spec.rb

```diff
@@ -47,7 +47,7 @@
       let(:inv_year_form) { create(:form1095_b, veteran_icn: '654678976543678', tax_year: 2008) }
 
       it 'fails if no template PDF for the tax_year' do
-        expect { inv_year_form.pdf_file }.to raise_error(RuntimeError, /1095-B for tax year 2008 not supported/)
+        expect { inv_year_form.pdf_file }.to raise_error(Common::Exceptions::UnprocessableEntity)
```

Comment:
Unfortunately, due to the way this error class is structured, it's very difficult to test the detail and source messages. IMO this is an issue with the error class.